src/Microsoft.AspNetCore.Server.IISIntegration/Microsoft.AspNetCore.Server.IISIntegration.csproj

```diff
@@ -1,23 +1,31 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>ASP.NET Core components for working with the IIS AspNetCoreModule.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;iis</PackageTags>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Numerics.Vectors" />
```

Comment:
I'm not sure need to flatten these (or that we need them all but we can remove these later)

Reply:
Kestrel does the same thing: https://github.com/aspnet/KestrelHttpServer/blob/26f1d4baa3a2f98edbf1d6f5996cf07dfc0693cc/src/Protocols.Abstractions/Protocols.Abstractions.csproj

Reply:
Yea, it was to work around a versioning issue that likely no longer exists. Lets file an issue to clean this up.